api/src/main/java/org/openmrs/module/registrationcore/api/RegistrationCoreService.java

```diff
@@ -74,15 +77,25 @@
      */
     public Patient registerPatient(Patient patient, List<Relationship> relationships, String identifierString, Location identifierLocation);
 
-    /**
-     * Registers patient and saves them in the database.
-     * @return the created patient
-     * @should create a patient from record with relationships
-     * @should fire an event when a patient is registered
-     * @should set wasPerson field to true for an existing person on the registration event
-     * @should fail if identifier does not pass validation
-     */
-    public Patient registerPatient(RegistrationData registrationData);
+	/**
+	 * Creates patient and saves them and their fingerprints in the database,
+	 * setting their identifier as specified instead of assigning automatically
+	 *
+	 * @return the created patient
+	 */
+
+	public Patient registerPatient(Patient patient, List<Relationship> relationships, String identifierString,
+								   Location identifierLocation, BiometricData biometricData);
+
+	/**
+	 * Registers patient and saves them in the database.
```

Comment:
It is always good to put a line of space after the method description.

Reply:
changed